features/cellular/framework/AT/ATHandler.cpp

```diff
@@ -58,7 +59,7 @@ static const uint8_t map_3gpp_errors[][2] =  {
     { 146, 46 }, { 178, 65 }, { 179, 66 }, { 180, 48 }, { 181, 83 }, { 171, 49 },
 };
 
-ATHandler::ATHandler(FileHandle *fh, EventQueue &queue, int timeout, const char *output_delimiter) :
+ATHandler::ATHandler(FileHandle *fh, EventQueue &queue, int timeout, const char *output_delimiter, uint16_t send_delay) :
```

Comment:
`uint16_t send_delay` - why uint16_t  was chosen? vs `int timeout` (3rd parameter)

Reply:
the delay is always positive, and 16 bits is enough for it for practical cases

Reply:
OK. I was just asking as basic types should be sufficient , if there was a specific reason to use uint16_t (why then timeout is plain `int`). If I add a new method here, I would be confused.